docs/tutorials/quickstart.rst

```diff
@@ -35,8 +35,10 @@ Now paste the following snippet into your ``python`` REPL:
 
 You're supposed to see ``Hello Orquestra!`` text in your terminal output.
 
-Running in process executes the workflow in a single thread.
+Running with ``in_process`` executes the workflow in a single thread.
 This means that the ``run()`` command blocks execution until the workflow run completes, and that results from the run will not persist if the process is terminated.
 
 The ``in_process`` execution is similar to directly calling your function in Python, and it's suitable for quick prototypes or debugging, but it doesn't show the full potential of Orquestra.
 For more advanced features, see the next tutorials in this series.
+
+.. [1] The use of the name ``workflow_defs.py`` is not a requirement. Rather, this is a convenient convention that we adopt throughout this documentation.
```

Comment:
```suggestion
.. [1] The use of the name ``workflow_defs.py`` is not a requirement anymore. Nowadays, this is a convenient convention that we adopt throughout this documentation.
```

Reply:
I think I'd prefer to keep this information framed in terms of "this is how things are" rather than expressing it as a change. How would you feel about

```suggestion
.. [1] The use of the name ``workflow_defs.py`` is not a requirement. Rather, this is a convenient convention that we adopt throughout this documentation. Note that some early versions of the SDK did enforce this name, but this restriction has since been lifted.
```
?